Source/Scene/ModelExperimental/buildDrawCommands.js

```diff
@@ -182,9 +160,10 @@ function deriveTranslucentCommand(command) {
   return derivedCommand;
 }
 
-function getIndexBuffer(primitiveRenderResources, debugWireframe, frameState) {
-  if (debugWireframe) {
-    return createWireframeIndexBuffer(primitiveRenderResources, frameState);
+function getIndexBuffer(primitiveRenderResources, frameState) {
```

Comment:
does this need a `/** @private */` comment on top of it? looks like I forgot to add one in my PR